cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/ApplyController.java

```diff
@@ -55,4 +56,32 @@ public Integer call() throws Exception {
             return 0;
         }
     }
+
+    @CommandLine.Command(name = "services", description = "Create/remove services that are present in the given yaml" +
```

Comment:
you can use static import, so that you will only need the "Command", without the "CommandLine".